web/static/js/components/import_export_tray.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+class ImportExportTray extends React.Component {
```

Comment:
this is the tray.

during export: it receives clean data from the editor and does all the serialization and data tweaks here. 
during import: validates pasted data and sets the passes the clean parsed data back to the editor

has a loading screen